### DISCUSSION TITLE
Github Copilot extension expect bower.json file to exist

### DISCUSSION THREAD

**Discussion by coolemur** [community/community#46791] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hello,

I am getting this message from GC plugin each time I open my vscode:

`GitHub Copilot could not connect to server. Extension activation failed: "cannot open file:///.../bower.json. Detail: Unable to read file '/.../bower.json' (Error: Unable to resolve nonexistent file '/.../bower.json')"`

I have tried other suggested methods related to "could not connect to server" problem: reinstalling copilot plugin, signing out and in again, disabling/re-enabling plugin, etc. Nothing worked so far.

I even tried to create a very clean "project" which has only npm init - package.json file in it, issue still remains when I open this folder.

When I create bower.json file in folder, error message disappears, but I don't prefer to have additional file in my folders.

Anyone knows why is this happening or have any suggestions what I could try out ?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @coolemur ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
